nautobot_circuit_maintenance/navigation.py

```diff
@@ -3,6 +3,11 @@
 from nautobot.utilities.choices import ButtonColorChoices
 
 menu_items = (
+    PluginMenuItem(
+        link="plugins:nautobot_circuit_maintenance:circuitmaintenance_overview",
+        link_text="Circuit Maintenance Dashboard",
```

Comment:
what about?

```suggestion
        link_text="Dashboard",
```

Reply:
`CM Dashboard` work to put a small specific to it?

Reply:
being within the "context" of "Circuit Maintenance", it seems redundant to me. Why do you think is relevant to be specific about it?

Reply:
In my design preferences I like to see the name of it to be a bit more explicit about what dashboard I'm going to. Let's say there are 3 plugins inside of circuits, and all of them have "Dashboard" for an option. This can add confusion about which dashboard in particular it is that is being referred to if you miss the heading section.

Reply:
Updated regardless.

Reply:
I'm not strong on this point, just trying to get more inputs